lib/config/Configuration.js

```diff
@@ -31,13 +34,24 @@ class Configuration {
 		return this.#mavenSnapshotEndpointUrl;
 	}
 
+	/**
+	 * Configurable directory where the artifacts would be stored.
```

Comment:
what do we define as "artefacts"? these are the framework libs isn'´t?

Reply:
By artifacts, I mean everything. It's basically the `.ui5` directory whith the following structure inside:
```yaml
- framework
   - artifacts
   - locks
   - metadata
   - packages
   - staging
```

Maybe, a better wording should be chosen? 🤔

Reply:
i would propose "framework artefacts"

Reply:
Thanks!